js/index.js

```diff
@@ -94,8 +94,12 @@ function jobs(id) {
   } else {
     x.style.display = "none";
   }
+ 
 } 
-
+ /* how to disable the button when the page is refreshed? 
+window.onload = function() {
+  document.getElementById("jobs").disabled = true;
+*/
```

Comment:
I'm unclear on which button you're wanting to disable. Please message me on Slack with more info and we can work through it there.